src/spamfilter.c

```diff
@@ -532,17 +532,17 @@ void stripall(char new[512], char *org)
         {
             // Color codes: 1-2 digits, then optionally followed by a comma and an additional 1-2 digits
             org++;
-            if(*org && IsDigit(*org))
```

Comment:
Why remove the sanity checks on *org?  Is it covered elsewhere that we haven't overrun this string?